src/normed_groups2.tex

```diff
@@ -52,15 +52,15 @@ \subsection{Variants of normed groups}
   Then
   \[ X_{\leq c} := \varprojlim_i X_{i,\leq c} \]
   is compact Hausdorff, and
-  \[ X=\bigcup_c X_{\leq c} \]
+  \[ X=\varinjlim_c X_{\leq c} \]
   is naturally a compact-Hausdorffly-filtered pseudonormed abelian group
-  which is the limit of $(X_i)_i$ in the strict category structure.
+  which is the colimit of $(X_i)_i$ in the strict category structure.
```

Comment:
I think this should actually stay `limit`. It's refering to line 53, right?

Reply:
Right!